velox/dwio/parquet/reader/ParquetReader.cpp

```diff
@@ -140,14 +140,42 @@ void toDuckDbFilter(
       }
       break;
     }
+    case common::FilterKind::kBigintValuesUsingHashTable: {
```

Comment:
It would be nice to add a small test to velox/dwio/parquet/tests/ParquetReaderTest.cpp